sqlserver/tests/test_integration.py

```diff
@@ -581,3 +582,44 @@ def test_resolved_hostname_set(
     aggregator.assert_metric_has_tag(
         "sqlserver.stats.batch_requests", "dd.internal.resource:database_instance:{}".format(expected_hostname)
     )
+
+
+@pytest.mark.parametrize(
+    'dbm_enabled, reported_hostname',
+    [
+        (True, None),
+        (False, None),
+        (True, 'forced_hostname'),
+        (True, 'forced_hostname'),
+    ],
+)
+@pytest.mark.integration
+@pytest.mark.usefixtures('dd_environment')
+def test_database_instance_metadata(aggregator, dd_run_check, instance_docker, dbm_enabled, reported_hostname):
+    instance_docker['dbm'] = dbm_enabled
+    if reported_hostname:
+        instance_docker['reported_hostname'] = reported_hostname
+    expected_host = reported_hostname if reported_hostname else 'stubbed.hostname'
+    mysql_check = SQLServer(CHECK_NAME, {}, [instance_docker])
```

Comment:
Nit: `sqlserver_check`

Reply:
Ah yeah, thanks for catching that. It's fixed now 🙇 .